crates/core/tedge_agent/src/tedge_operation_converter/actor.rs

```diff
@@ -96,26 +109,45 @@ impl TedgeOperationConverterActor {
                     Err(err) => error!("Incorrect software update request payload: {err}"),
                 }
             }
-            "tedge/commands/req/control/restart" => {
-                match RestartOperationRequest::from_slice(message.payload_bytes()) {
-                    Ok(request) => {
-                        self.restart_sender.send(request).await?;
-                    }
-                    Err(err) => error!("Incorrect restart request payload: {err}"),
+            _ => {
+                // Not a tedge/commands !
+            }
+        }
+
+        if message.topic.name.as_str().starts_with("tedge") {
+            return Ok(());
```

Comment:
No this is temporary code. The first part of the method is related to `tedge` topics, the second part to `te` topics.

I could have write this differently, but came with this not so nice return to avoid confusing diffs in the PR.